app/src/main/java/com/hammer/talkbbokki/presentation/navigation/TalkbbokkiNavHost.kt

```diff
@@ -73,53 +74,55 @@ fun TalkbbokkiNavHost(
                 navController.navigate(OnBoardingDestination.route) {
                     popUpTo(MainDestination.route) { inclusive = true }
                 }
-            }
+            },
         )
         topicListGraph(
             navigateToDetail = { item ->
                 navController.navigate(DetailDestination.route + "?topic=$item")
             },
             navigateToMain = {
                 navController.popBackStack()
-            }
+            },
         )
         detailGraph(
             navigateToTopicList = {
                 navController.popBackStack()
-            }
-        )
-        eventGraph(
-            navigateToComments = {},
-            navigateToMain = {
-                navController.popBackStack()
-            }
+            },
         )
         eventGraph(
-            navigateToComments = { topicId ->
-                navController.navigate(CommentsDestination.route + "?topicId=$topicId")
+            navigateToComments = { topicId, commentCount ->
+                navController.navigate(CommentsDestination.route + "?topicId=$topicId&commentCount=$commentCount")
```

Comment:
🚫 **[ktlint]** <standard:argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)